### DISCUSSION TITLE
Privacy concerns

### DISCUSSION THREAD

**Discussion by viasux** [tauri-apps/tauri#4089] [General]
Why is microsoft edge used when on windows? Doesn't that kind of contradict the seeming purpose of this as an alternative for electron? Edge is just chromium but with more spyware, so why develop for it?

**Comment by JonasKruckenberg** [comment]
Tauri doesn't bundle chromium like electron does, it uses the preinstalled component that is shipped with every windows distro (optional on win10, but installed by default on win11) this is the reason we rely on it.

It's also worth noting that we don't use edge, we use Webview2 which (despite their marketing material claiming otherwise) is not the browser directly, but rather a component similar and most likely built from the same codebase. This is equivalent to macOS where Safari and WKWebkit are very closely related, but still technically different things.

Regarding the privacy of Webview2, [here](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1059#issuecomment-796348416) is an issue about potential privacy issues. The main takeaway is:
WebView2 is a windows component like, File Explorer, Powershell or the Win32 API and is governed by the exact same privacy rules. This means users can toggle privacy settings globally for the whole OS and WebView2 will respect that setting. 

[Here](https://github.com/MicrosoftEdge/WebView2Feedback/issues/907) is another issue (actually raised by @nothingismagick who founded this project) talking about the legal and privacy issues associated with WebView2.

Main takeaway: WebView2 is the only available webview on windows, if you or your users are concerned about that, they can disable telemetry and tracking OS-wide.

**Reply by FabianLars** [reply]
On that note: Microsoft doesn't need WebView2 to spy on you or your users if they own the whole OS. That doesn't "justify" any privacy concerns with WV2 of course, but it's still something you have to have in mind when thinking about privacy.

P.S. Just wanna highlight that part of Jonas' comment:
> WebView2 is the only available webview on windows.

**Reply by viasux** [reply]
Is webview2 open source?

**Reply by FabianLars** [reply]
Nope

**Reply by viasux** [reply]
> On that note: Microsoft doesn't need WebView2 to spy on you or your users if they own the whole OS. That doesn't "justify" any privacy concerns with WV2 of course, but it's still something you have to have in mind when thinking about privacy.
> 
> P.S. Just wanna highlight that part of Jonas' comment:
> > WebView2 is the only available webview on windows.

I use windows out of necessity, and only use win10 AME.
